app/views/instruments/_instrument_fields.html.haml

```diff
@@ -22,7 +22,11 @@
     = rf.input :port
     = rf.input :username
     = rf.input :password, as: :string
-    = rf.input :auto_logout, :as => :boolean, :label => false, :inline_label => t('instruments.instrument_fields.relay.label.auto_logout')
+    = rf.input :auto_logout do
+      = rf.check_box :auto_logout
```

Comment:
I tried, but it keeps adding a label even though I'm specifying `label: false`. That's throwing the style off and pushing the text to the another line.